src/server/routes/consents.ts

```diff
@@ -168,44 +171,40 @@ export const consentPages: ConsentPage[] = [
         throw error;
       }
     },
-    update: async (ip, sc_gu_u, body, geo) => {
-      const newslettersPage = NewsletterMap.get(geo) as string[];
-
+    update: async (ip, sc_gu_u, body) => {
```

Comment:
can we update the caller to stop passing in `geo`?

Reply:
👍 removed the `geo` in all the callers for `update`